src/components/elements/InputText/InputText.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```

Comment:
I think this component is fine, we can always visit this later my only ask is rename it to just `Input` not `InputText`